src/style.css

```diff
@@ -0,0 +1,144 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  height: 406px;
+  width: 200px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  --min-gap: 16px;
+}
+
+.product-card__computer-view {
+  margin: 32px 19px 40px;
+}
+
+.view {
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__computer-name {
+  margin: 0 var(--min-gap) 4px;
+}
+
+.name {
+  height: 36px;
+  width: 168px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  color: #060b35;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-card__computer-cod {
+  margin: 0 90px 16px var(--min-gap);
+}
+
+.cod {
+  height: 14px;
+  width: 94px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.rating {
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
+  margin: 0 16px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  display: inline-block;
+}
+
+.stars :nth-child(n+5) {
+  background-image: url(images/star.svg);
+}
+
+.reviews_number {
+  height: 16px;
+  width: 53px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+}
+
+.prise {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px var(--min-gap) 0;
+}
+
+.price-name {
+  height: 18px;
+  width: 32px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.price-value {
+  height: 18px;
+  width: 79px;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  color: #060b35;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.button {
+  margin: 16px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  height: 40px;
+  width: 166px;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  color: #ffff;
+  font-size: 14px;
+  line-height: 16px;
+  outline: none;
+  border: none;
+  padding: 0;
+  text-transform: uppercase;
+}
+
+.button:hover {
+  margin: var(--min-gap);
+  background-color: #fff;
+  border-radius: 5px;
+  height: 40px;
+  width: 166px;
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  color: #00acdc;
+  font-size: 14px;
+  line-height: 16px;
+  outline: none;
+  border: 1px solid #00acdc;
+  padding: 0;
+  text-transform: uppercase;
+}
```

Comment:
[STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover

leave only colors here